### PR TITLE
Cleanup and increase readability

### PR DESCRIPTION
**Pull request by fabian-bizfactory**

### CODE REVIEW

File: frontend/src/js/preview-v2/Charts.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const DirectionSelector = styled("div")`
 `;
 
 const SxIconButton = styled(IconButton)`
-  font-size: 24px;
+  font-size: 2Diagrams_PER_PAGEpx;
```
Comment (MarcoKorinth) [inline frontend/src/js/preview-v2/Charts.tsx]: that can't be right.
Replace all error?

File: frontend/src/js/preview-v2/HeadlineStats.tsx [hunk 2]
```diff
@@ -52,26 +40,25 @@ export default function HeadlineStats({ statistics }: HeadlineStatsProps) {
...
-  const { total, dateRange } = statistics;
-
+  const dateRange = statistics?.dateRange;
+  
   return (
     <Root>
       <MetaValue>
         <Key>Zeilen:</Key>
-        {total}
+        {statistics?.total || "Noch nicht geladen"}
```
Comment (MarcoKorinth) [inline frontend/src/js/preview-v2/HeadlineStats.tsx:49]: maybe find a i18next string for that.
Now that I think about it, we should consider extracting all the strings (Zeilen, Min Datum, etc.) to i18next language files.
What do you think?

**Review by MarcoKorinth — CHANGES_REQUESTED** [review]
found a syntax error and one more thing about the strings we used

File: frontend/src/js/preview-v2/HeadlineStats.tsx (hunk 2 above)
Comment (fabian-bizfactory) [reply frontend/src/js/preview-v2/HeadlineStats.tsx:49]: We will recode the Headline Stats in the next Step - so I don't think that is necessary rn

**Review by MarcoKorinth — APPROVED** [review]

**Pull request merged**